### PR TITLE
SPLICE-2298 SerDe dataTypeDescriptors for building template row in StatisticsOperation

### PR DESCRIPTION
**Pull request by changli6**

### CODE REVIEW

**Comment by changli6** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by changli6** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Comment by changli6** [comment]
Verification steps please referencing Jira.

File: splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsOperation.java [hunk 1]
```diff
@@ -215,4 +231,18 @@ public double getSampleFraction() {
...
+    private ExecRow buildTemplateRow(DataTypeDescriptor[] dataTypeDescriptors)  throws StandardException{
+        ExecRow row = new ValueRow(dataTypeDescriptors.length);
+        int outputCol = 0;
+        for (DataTypeDescriptor dtd : dataTypeDescriptors) {
+            row.setColumn(outputCol + 1, dtd.getNull());
+            if (row.getColumn(outputCol + 1 ) instanceof SQLDecimal) {
+                ((SQLDecimal) row.getColumn(outputCol + 1)).setPrecision(dtd.getPrecision());
+                ((SQLDecimal) row.getColumn(outputCol + 1)).setScale(dtd.getScale());
+            }
+            outputCol++;
```
Comment (jyuanca) [inline splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsOperation.java]: The null SQLDecimal should have precision and scale, and you don't need to set it again

Comment (changli6) [reply splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsOperation.java]: Good catch, thanks @jyuanca .

**Comment by changli6** [comment]
jenkins please test branch @cdh5.12.2

**Pull request merged**